frontend/console/src/components/StatusIndicator.tsx

```diff
@@ -0,0 +1,24 @@
+import type React from 'react'
+import { classNames } from '../utils'
+
+type StatusIndicatorProps = {
+  state: 'success' | 'error' | 'idle'
+  text?: string
+}
+
+export const StatusIndicator: React.FC<StatusIndicatorProps> = ({ state, text }) => {
+  const backgrounds = {
+    idle: 'text-gray-500 bg-gray-100/10',
+    success: 'text-green-400 bg-green-400/10',
+    error: 'text-rose-400 bg-rose-400/10',
```

Comment:
I feel like these also look great on dark mode but feel a little light for light mode. What do you think?

Reply:
Good call! Updated.